TwoP/DatePickerTextField.swift

```diff
@@ -44,7 +44,8 @@ class DatePickerTextField: UITextField {
     func sharedInit() {
         style = .time
         inputView = picker
-        addTarget(self, action: #selector(datePickerDateDidChange(picker:)), for: .valueChanged)
+        picker.backgroundColor = UIColor.white
+        picker.addTarget(self, action: #selector(datePickerDateDidChange(picker:)), for: .valueChanged)
     }
```

Comment:
• changed picker bg color to white
• fixed addTarget so it's adding to the picker, not the textField